librad/src/git/types.rs

```diff
@@ -108,69 +100,59 @@ impl<R> SymbolicRef<R> {
     ///     is passed.
     pub fn create<'a>(&self, repo: &'a git2::Repository) -> Result<git2::Reference<'a>, git2::Error>
     where
-        R: Display + Clone,
+        S: Display,
```

Comment:
Pretty weird that it can be _any_ `S` that implements `Display` but I guess this makes it easier to use ergonomically. Wondering if we should seal it with a `trait Ref: Sealed` or something?